packages/kbn-investigation-shared/src/rest_specs/create_item.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+import { investigationItemsSchema } from '../schema';
+import { investigationItemResponseSchema } from './investigation_item';
+
+const createInvestigationItemParamsSchema = t.type({
+  path: t.type({
+    investigationId: t.string,
+  }),
+  body: investigationItemsSchema,
```

Comment:
Is it here `investigationItemsSchema` or maybe `investigationItemSchema`

Reply:
Oh right! It's either or, somehow I confused it and thought it is an array of all types of items. Makes total sense now